_inc/lib/class.media.php

```diff
@@ -242,15 +235,15 @@ public static function get_snapshot( $media_item ) {
 			'file'             => (string) $file_paths['basename'],
 			'extension'        => (string) $file_paths['extension'],
 			'mime_type'        => (string) $media_item->post_mime_type,
-			'size'             => (int) filesize( $current_file ) 
+			'size'             => (int) filesize( $current_file )
```

Comment:
Since we are touching that line...

```suggestion
			'size'             => (int) filesize( $current_file ),
```